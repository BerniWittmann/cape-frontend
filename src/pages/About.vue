<template>
  <v-layout>
    <div class="about">
      <h1>{{ $t('about.title') }}</h1>
    </div>
  </v-layout>
</template>

<script>
import DefaultLayout from '@/layouts/Default.vue'

export default {
  components: {
    VLayout: DefaultLayout
  }
}
</script>

<style scoped lang="scss">
.about {
  text-align: center;
}
</style>
