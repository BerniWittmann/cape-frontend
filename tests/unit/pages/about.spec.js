import { shallowMount } from '@vue/test-utils'
import { i18n } from '../setupPlugins'

import About from '@/pages/About.vue'

import EmptySlotComponent from '../EmptySlotComponent.vue'

describe('Pages', () => {
  describe('About', () => {
    it('renders', () => {
      const wrapper = shallowMount(About, {
        i18n,
        stubs: {
          'v-layout': EmptySlotComponent
        }
      })
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
