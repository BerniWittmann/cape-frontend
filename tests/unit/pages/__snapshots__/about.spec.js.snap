// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages About renders 1`] = `
<div>
  <div class="about">
    <h1>about.title</h1>
  </div>
</div>
`;
